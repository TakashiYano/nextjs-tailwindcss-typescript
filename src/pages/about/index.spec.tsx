import renderer from "react-test-renderer";

import About from "./index.page";

describe("About page", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
