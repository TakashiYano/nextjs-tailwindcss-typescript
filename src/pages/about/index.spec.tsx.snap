// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<h2>
  About
</h2>
`;